src/metadataTextTracks.js

```diff
@@ -0,0 +1,123 @@
+import videojs from 'video.js';
```

Comment:
I'll add a descriptive comment at the top of the file.

The main reason for me going with "metadataTextTracks" was because there are two main methods in this file: process(on general metadata text tracks) and processAdTrack(on specifically ad text tracks). Any suggestions that encompass both would be appreciated! Perhaps `cueTextTracks.js`?